tests/gui/measure/test_hardwarebase.py

```diff
@@ -0,0 +1,156 @@
+"""Tests for module hardwarebase of viperleed.gui.measure."""
+
+__authors__ = (
+    'Michele Riva (@michele-riva)',
+    'Florian Dörr (@FlorianDoerr)',
+    )
+__copyright__ = 'Copyright (c) 2019-2025 ViPErLEED developers'
+__created__ = '2025-10-13'
+__license__ = 'GPLv3+'
+
+import abc
+import sys
+import types
+import pytest
+
+from viperleed.gui.measure.hardwarebase import class_from_name
+from viperleed.gui.measure.hardwarebase import get_devices
+from viperleed.gui.measure.hardwarebase import import_with_sub_modules
+
+
+@pytest.fixture
+def fake_pkg(mocker):
```

Comment:
```suggestion
@pytest.fixture(name='fake_pkg')
def fixture_fake_pkg(mocker):
```

This is to prevent pylint complaints about argument names below shadowing the fake_pkg module-level name.